automatic/openssl.light/tools/chocolateyInstall.ps1

```diff
@@ -20,15 +17,10 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+Remove-Item -Force -ea 0 "$toolsPath\*.exe","$toolsPath\*.ignore"
 
 $installLocation = Get-AppInstallLocation $packageArgs.softwareName
-$binPath = "$($installLocation)\bin" 
-
-if(Test-Path $installLocation ) {
-  Install-ChocolateyPath -PathToInstall $binPath -PathType Machine 
-}
-else {
-  Write-Warning "Could not add install directory to path"
-}
+if (!$installLocation)  { Write-Warning "Can't find install location, PATH not updated"; return }
+Write-Host "Installed to '$installLocation'"
```

Comment:
shouldn't this be added in an `else` statement?
What is the point of outputting the install location if it isn't found?

Reply:
Nah, it exists the script in if part with `return` since there is nothing to be done

Reply:
ah right, sorry missed the return statement